plugins/source/mixpanel/client/spec.go

```diff
@@ -0,0 +1,17 @@
+package client
+
+type Spec struct {
+	Username string `json:"username,omitempty"`
+	Secret   string `json:"secret,omitempty"`
+
+	ProjectID   int64 `json:"project_id,omitempty"`
+	WorkspaceID int64 `json:"workspace_id,omitempty"`
+
+	StartDate string `json:"start_date,omitempty"`
```

Comment:
I guess these are optional as well, because we provide defaults (comment below seems to indicate that they are not)